lib/ansible/module_utils/basic.py

```diff
@@ -80,7 +80,7 @@
 import platform
 import errno
 import datetime
-from itertools import repeat, chain
+from itertools import repeat, chain, imap
```

Comment:
Yes, I have seen this. It will be fixed Monday ; and I will provide a full test scenario.